### PR TITLE
Add go to theme check page link on plugin page

### PR DESCRIPTION
**Pull request by rinkuyadav999**

### CODE REVIEW

File: ns-theme-check.php [hunk 1]
```diff
@@ -27,3 +27,18 @@
 
 // Load admin.
 require_once NS_THEME_CHECK_DIR . '/inc/admin.php';
+
+/**
+ * Add go to theme check page link on plugin page.
+ *
+ * @since 0.1.2
```
Comment (grappler) [inline ns-theme-check.php]: This would be `0.1.3` because that is the next version.

File: ns-theme-check.php [hunk 2]
```diff
@@ -27,3 +27,18 @@
...
+ */
+
```
Comment (grappler) [inline ns-theme-check.php]: I would remove the empty space.

File: ns-theme-check.php [hunk 3]
```diff
@@ -27,3 +27,18 @@
...
+function ns_theme_check_plugin_settings_link( $links ) { 
+  $settings_link = '<a href="themes.php?page=ns-theme-check">' . esc_attr( 'Theme Check Page', 'ns-theme-check' ) . '</a>';
```
Comment (grappler) [inline ns-theme-check.php]: Would be better to use tabs not spaces. I would name the variable `$theme_check_link`.

File: ns-theme-check.php [hunk 4]
```diff
@@ -27,3 +27,18 @@
...
+
+/**
+ * Add go to theme check page link on plugin page.
+ *
+ * @since 0.1.2
+ */
+
+function ns_theme_check_plugin_settings_link( $links ) { 
+  $settings_link = '<a href="themes.php?page=ns-theme-check">' . esc_attr( 'Theme Check Page', 'ns-theme-check' ) . '</a>'; 
+  array_unshift($links, $settings_link);
```
Comment (grappler) [inline ns-theme-check.php]: Would be good to follow the WP coding standards and add a space before or after the brackets.

**Comment by grappler** [comment]
This seems like a good idea.

**Comment by rinkuyadav999** [comment]
New PR.

**Comment by grappler** [comment]
@rinkuyadav999 You don't need to create a new PR. You can just commit in the same branch that you have.

**Comment by rinkuyadav999** [comment]
Is it good to go?

File: ns-theme-check.php [hunk 5]
```diff
@@ -27,3 +27,17 @@
...
 // Load admin.
 require_once NS_THEME_CHECK_DIR . '/inc/admin.php';
+
+/**
+ * Add go to theme check page link on plugin page.
+ *
+ * @since 0.1.3
+ */
+function ns_theme_check_plugin_settings_link( $links ) {
+	$theme_check_link = '<a href="themes.php?page=ns-theme-check">' . esc_attr( 'Theme Check Page', 'ns-theme-check' ) . '</a>';
```
Comment (ernilambar) [inline ns-theme-check.php]: `esc_attr__` should be used here instead of `esc_attr`.

Comment (ernilambar) [inline ns-theme-check.php]: @rinkuyadav999 ^

Comment (rinkuyadav999) [inline ns-theme-check.php]: :)

**Comment by rinkuyadav999** [comment]
@ernilambar seems ready to fly.

**Pull request merged**